### PR TITLE
Code formatting

### PR DESCRIPTION
**Pull request by peppy**
Getting automated code formatting checks setup.

May take a few tries.

### CODE REVIEW

**Comment by SirCmpwn** [comment]
Something I notably dislike about osu code: the use of @ strings everywhere.

**Comment by SirCmpwn** [comment]
I prefer the opposite, marking strings that *do* require localization. However, I'm okay with @ strings if that has semantic meaning, I thought they were just unnecessary.

**Comment by peppy** [comment]
Anyways, for the time being i'm going to plow forward with applying all resharper suggestions that make sense so we have the code in some sense of conformity. We can discuss individual changes once that is complete.

**Comment by peppy** [comment]
Probably a good checkpoint to merge these changes into master. Not all warnings are fixed, but the remaining warnings are resharper/mono level (and therefore only seem to fail CI on travis)

File: osu.Framework/Graphics/Sprites/SpriteCircular.cs [hunk 1]
```diff
@@ -17,15 +17,10 @@ public class SpriteCircular : Sprite
...
-            : base(texture)
-        {
-            HoverRadius = radius > 0 ? radius : texture.DisplayWidth / 2f;
-        }
-
         internal override bool Contains(Vector2 screenSpacePos)
         {
-            Vector2 screenHoverRadius = new Vector2(HoverRadius, HoverRadius) * DrawInfo.Matrix.ExtractScale().Xy;
+            float hoverRadius = HoverRadius > 0 ? HoverRadius : Texture.DisplayWidth / 2f;
+            Vector2 screenHoverRadius = new Vector2(hoverRadius, hoverRadius) * DrawInfo.Matrix.ExtractScale().Xy;
```
Comment (Tom94) [inline osu.Framework/Graphics/Sprites/SpriteCircular.cs]: I know this has nothing to do with this PR, but it would be better if you computed the DistanceSquared in object-space after applying `DrawInfo.MatrixInverse` to `screenSpacePos`. That way you wouldn't break ovals. :P

Comment (peppy) [inline osu.Framework/Graphics/Sprites/SpriteCircular.cs]: prolly best to do that in another PR
